responsibleai/tests/counterfactual/test_counterfactual_advanced_features.py

```diff
@@ -136,3 +150,37 @@ def test_counterfactual_manager_save_load(self, tmpdir):
         assert len(counterfactual_config_list) == 2
         assert counterfactual_config_list[0].explainer is not None
         assert counterfactual_config_list[1].explainer is not None
+
+    def test_counterfactual_manager_get_data(self):
+        X_train, X_test, y_train, y_test, feature_names, _ = \
+            create_iris_data()
+
+        model = create_lightgbm_classifier(X_train, y_train)
+        X_train['target'] = y_train
+        X_test['target'] = y_test
+
+        rai_insights = RAIInsights(
+            model=model,
+            train=X_train,
+            test=X_test.iloc[0:10],
+            target_column='target',
+            task_type='classification')
+
+        rai_insights.counterfactual.add(
+            total_CFs=10, desired_class=2,
+            features_to_vary=[feature_names[0]],
+            permitted_range={feature_names[0]: [2.0, 5.0]})
+        rai_insights.counterfactual.compute()
+
+        serialized_data = rai_insights.counterfactual.get_data()
+        assert serialized_data is not None
```

Comment:
nit: maybe extract the asserts into a utility function, as it seems like it could be re-used in other tests as well?

Reply:
Right now these asserts are only in this function. Maybe, we can move them into a function when other tests use these asserts.